src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/LaunchProfiles/SetLaunchProfilePropertyAction.cs

```diff
@@ -28,7 +28,7 @@ public async Task ReceiveResultAsync(QueryProcessResult<IEntityValue> result)
             result.Request.QueryExecutionContext.CancellationToken.ThrowIfCancellationRequested();
             if (((IEntityValueFromProvider)result.Result).ProviderState is ContextAndRuleProviderState state)
             {
-                var cache = state.Cache;
+                var cache = state.ProjectState;
```

Comment:
`state` here is a `ProviderState` and `cache` should probably be `state`, I'd reckon. [or something of the sort]